source/operations/concepts.rst

```diff
@@ -8,126 +8,173 @@ Core Operational Concepts
-   :mc:`minio server` command represents one Server Pool:
+How does MinIO link multiple server pools into a single MinIO cluster?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A cluster refers to an entire MinIO deployment consisting of one or more Server Pools. 
 
-   .. code-block:: shell
+Consider the command below that creates a cluster consisting of two Server Pools, each with 4 :mc:`minio server` nodes and 4 drives per node for a total of 32 drives. 
 
-      minio server https://minio{1...4}.example.net/mnt/disk{1...4} \
-                   https://minio{5...8}.example.net/mnt/disk{1...4}
+.. code-block:: shell
+
+   minio server https://minio{1...4}.example.net/mnt/disk{1...4} \
+                https://minio{5...8}.example.net/mnt/disk{1...4}
                    
-                   |                    Server Pool                |
+                |                    Server Pool                |
+   
+Within a cluster, MinIO always stores each unique object and all versions of that object on the same Server Pool.
+
+MinIO strongly recommends production clusters consist of a *minimum* of 4 :mc:`minio server` nodes in a Server Pool for proper high availability and durability guarantees.
+
+Can I change the size of an existing MinIO deployment?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can move objects from an existing server pool on to other server pools on the deployment to reduce the size of a deployment by decommissioning a server pool.
+Likewise, you can add on a new server pool to an existing deployment to expand the total size of a deployment with a horizontal expansion.
+
+When the time comes to retire or replace a server pool, :ref:`decommissioning <minio-decommissioning>` is the process of draining the objects of a pool to other active pools on the deployment.
+MinIO rewrites objects from the decommissioned pool and does not allow additional writes to the pool while it is in decommissioning.
    
-   The above example describes two Server Pools, each consisting of 4
-   :mc:`minio server` nodes with 4 drives each for a total of 32 drives. MinIO 
-   always stores each unique object and all versions of that object on the 
-   same Server Pool.
+Once started, decommissioning cannot be stopped.
+
+In a distributed MinIO deployment, you can upgrade the total available size of a :ref:`deployment <expand-minio-distributed>` or :ref:`cluster <minio-k8s-expand-minio-tenant>` by adding one or more additional server pools.
+The addition of a server pool is an expansion.
+
+Does MinIO provide a tool to manage multiple MinIO clusters within a single interface?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In Kubernetes, a tenant is a specific instance of MinIO with its own node(s), buckets, objects, volumes, access keys, and lifecycle management rules.
+The MinIO Operator can manage many MinIO tenants.
```

Comment:
I think we need to revisit this - Kubernetes is not a tool we provide. And `mc` can technically speak to multiple MinIO clusters.

Probably :+1: 

"How do I manage a MinIO cluster?"

- using mc
- using console
- for Kubernetes, using MinIO Operator Console.

Reply:
Good point. Updated to a linked list as suggested.